addons/skin.estuary/1080i/Defaults.xml

```diff
@@ -151,9 +151,10 @@
 		<aligny>center</aligny>
 		<texturefocus border="40" colordiffuse="button_focus">buttons/button-fo.png</texturefocus>
 		<texturenofocus border="40">buttons/button-nofo.png</texturenofocus>
+		<orientation>horizontal</orientation>
 		<sliderwidth>150</sliderwidth>
 		<sliderheight>28</sliderheight>
```

Comment:
@ronie @phil65 
How can we provide default values for `sliderwidth` and `sliderheight` for horizontal and vertical sliders? Or should we remove them like we did for GUISliders?